### PR TITLE
feat: replace goerli with sepolia

### PR DESCRIPTION
**Pull request by lauti7**
this pr shouldn't be merged until we proceed with the migration

### CODE REVIEW

File: packages/@dcl/sdk-commands/src/commands/deploy/index.ts
Comment (fzavalia) [inline packages/@dcl/sdk-commands/src/commands/deploy/index.ts:138]: In this cases that is easy to support both testnets I would just do so instead of completely removing Goerli

```suggestion
  const network = chainId === ChainId.ETHEREUM_GOERLI ? 'goerli' : chainId === ChainId.ETHEREUM_SEPOLIA ? 'sepolia' : 'mainnet'
```

**Review by fzavalia — CHANGES_REQUESTED** [review]

**Review by fzavalia — COMMENTED** [review]
LGTM

Let's wait for the proper reviewer's approval

**Review by nicoecheza — CHANGES_REQUESTED** [review]

**Review by gonpombo8 — APPROVED** [review]

File: packages/@dcl/sdk-commands/package.json
Comment (gonpombo8) [inline packages/@dcl/sdk-commands/package.json:17]: I think you need to add the sepolia contracts and maybe some more config to the [linker-dapp](https://github.com/decentraland/linker-dapp) and bump it here too.

File: packages/@dcl/sdk-commands/src/commands/deploy/index.ts
Comment (gonpombo8) [inline packages/@dcl/sdk-commands/src/commands/deploy/index.ts:132]: are we going to still support goerli ? If it has an end we should remove all of it so we dont have death code here. (Just like we did with ropsten)
if we want to support the two testnets for a while, we can add an issue and remove it later.

File: test/sdk-commands/commands/deploy/index.spec.ts
Comment (lauti7) [reply test/sdk-commands/commands/deploy/index.spec.ts:20]: it doesn't compile without casting

File: packages/@dcl/sdk-commands/src/commands/deploy/index.ts
Comment (lauti7) [reply packages/@dcl/sdk-commands/src/commands/deploy/index.ts]: it's like that due to gamejam, removed now

**Review by nicoecheza — APPROVED** [review]

**Review by fzavalia — APPROVED** [review]

**Pull request merged**
